Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -1712,7 +1711,7 @@ internal override void Validate()
     {
         base.Validate();
         if (string.IsNullOrWhiteSpace(UploadId))
-            throw new ArgumentNullException(nameof(UploadId) + " not assigned for PutObjectPart operation.");
+            throw new ArgumentNullException(nameof(UploadId), $"{nameof(UploadId)} not assigned for PutObjectPart operation.");
```

Comment:
The single argument constructor takes the argument name as the paramName. The second constructor parameter is paramName, message.

[`public ArgumentNullException (string? paramName);`](https://learn.microsoft.com/en-us/dotnet/api/system.argumentnullexception.-ctor?view=net-7.0#system-argumentnullexception-ctor(system-string)) vs

[`public ArgumentNullException (string? paramName, string? message)`](https://learn.microsoft.com/en-us/dotnet/api/system.argumentnullexception.-ctor?view=net-7.0#system-argumentnullexception-ctor(system-string-system-string))